lib/valueflow.h

```diff
@@ -175,6 +176,8 @@ namespace ValueFlow {
         /** Is this value passed as default parameter to the function? */
         bool defaultArg;
 
+        int indirect;
```

Comment:
I guess you should check `indirect` in `Value::operator==`

Reply:
Please update `ValueFlow::Value::infoString()' also somehow. I think it would make sense to show this info in the debug output.

I do not have a strong opinion about how it is written. Maybe `<uninit1>` or `<uninit *>`.